### PR TITLE
impr(quotes): add Hebrew quotes

### PR DESCRIPTION
**Pull request by ItsBenDov**
- [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.

"Violence is eroding the foundation of Israeli democracy. She must be condemned, denounced, isolated."

"Make you a rabbi, and buy you a friend, and he will judge the whole person to a rightful place - odd translation, sounds better in hebrew."

"Honesty is strength, values are strength, cunning and corruption bring us down, generosity and love are the strongest forces in the world"

"what? No way?!" a very famous quote in israel, again, sounds better in Hebrew.

"There will be nothing, because there is nothing"

Thank you!

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8473333369

File: frontend/static/quotes/hebrew.json [hunk 1]
```diff
@@ -186,6 +186,36 @@
...
+      "source": "בנימין נתניהו",
+      "length": 25,
+      "id": 31
+    },
+        {
+      "text": "מה? מה פתאום?!",
+      "source": "בנימין נתניהו",
+      "length": 14,
+      "id": 32
+    },
```
Comment (Miodec) [inline frontend/static/quotes/hebrew.json:201]: Way too short, please remove or use a longer version.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8542359962

**Review by Miodec — CHANGES_REQUESTED** [review]
Update short quotes with longer versions (quotes should be at least 60 characters)

Also please update translations.

**Review by Miodec — APPROVED** [review]

**Pull request merged**